### DISCUSSION TITLE
How to get ownership back after it was transferred to an account that switched to an organization?

### DISCUSSION THREAD

**Discussion by jpascasio2** [community/community#132688] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all,

So the problem is that my school gave ownership to a client account for a repo and the client then changed their account to an organization losing the rights to ALL repos. I still have access to the private repo but the client does not. 

**Where does the ownership go**? I'm guessing the school organization gets ownership back because I can't fork the repo or do anything with it since I'm just a collaborator. 

Another question is **how can I recover the ownership**? I now work for the client and need to give ownership to their personal account. I would also like to retain the git history on the repo.

**Comment by knarrff** [comment]
> Where does the ownership go?

When you still have access to the repo: isn't the URL part before the repository name the "entity" (user/org) that owns that repository? You might want to make sure to look at that in a browser: if you only look at a saved URL in a git checkout, maybe github is nice to redirect you to another URL which you might not notice there - you should in a browser.

> how can I recover the ownership

I guess that depends on who now owns it. Generally I would suspect the owner transferring the repo.

> git history

As long as you still have at least one git clone of that repo locally around, you still have all the git history saved within that.

**Reply by jpascasio2** [reply]
Hey thanks for the answer!

So from my understanding, the original owner (school organization) is that "entity" in the URL. Another question forms from that, why did the URL not change when ownership was transferred to the client? From your response does that mean the URL is supposed to change depending on who owns it?

So from a timeline it looked similar to this:

- School Organization makes project
- Once project is finished, School Organization transfers ownership to client ACCOUNT (not organization yet)
- At this point, the URL remains the same i.e. School-Organization/School_Project_1
- Client decides to make their account into an organization which loses access to all repos owned
- Client no longer has access to the school project

I was added as a **member** of that school organization when I was working on the project, but I don't have permissions to pass ownership. Overall I think I might have to send an e-mail to the school but I wanted that to be the last resort if there was other options.

Thanks for clearing up a lot of my questions! I will leave this open for more replies and sorry if it's a bit confusing.

**Reply by knarrff** [reply]
I do read on https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/converting-a-user-into-an-organization "_Any forks of private repositories made with the converted personal account will be deleted._", yet you do write "_I still have access to the private repo_". I can only imagine that the "_made with_" is meant literally here.

**Reply by knarrff** [reply]
Maybe one interesting question: who is the current owner of the new organization? When converting the user account into an organization, the client must have given another account as the new owner.
